todo_app/app.py

```diff
@@ -8,7 +11,17 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', headers=['id', 'status', 'title'], results=get_items())
+
+
+@app.route('/add', methods=['GET', 'POST'])
+def add():
+    if request.method == 'POST':
+        data = request.form['input']
+        add_item(data)
+        return redirect('/')
+    if request.method == 'GET':
```

Comment:
Having a separate page for adding items is fine, but it might make sense to split this out into two routes (even if they have the same URL, but different methods) to make this a bit cleaner, e.g. 
```
@app.route('/add', methods=['POST'])
def add():
    data = request.form['input']
    add_item(data)
    return redirect('/')

@app.route('/add', methods=['GET'])
def get_add_form():
    return render_template('add.html')
```